packages/journald-cloudwatch-logs/journald-cloudwatch-logs.conf

```diff
@@ -0,0 +1,4 @@
+log_group = "bottlerocket-host"
+state_file = "/var/lib/journald-cloudwatch-logs/state"
+log_priority = "debug"
```

Comment:
Debug has been used as an example to see if we are getting all the logs or not. Possible values are: 0,1,2,3,4,5,6,7 or one of the corresponding "emerg", "alert", "crit", "err", "warning", "notice", "info", "debug". When a single log level is specified, all messages with this log level or a lower (hence more important) log level are read and pushed to CloudWatch